src/actions/action-creators.test.js

```diff
@@ -3,11 +3,13 @@ import * as types from '../constants/action-types';
 import thunkMiddleware from 'redux-thunk';
 import configureMockStore from 'redux-mock-store';
 import panes from '../constants/pane-types';
+import { initializeCalculator } from '../lib/calculator';
 
 const middlewares = [thunkMiddleware];
 const mockStore = configureMockStore(middlewares);
 
 describe('Action creators', () => {
+  /***** sync *****/
```

Comment:
i'd remove this comment, the string inside of the describe in L13 lets us know we're talking about synchronous stuff

Reply:
Done